templates/provider/cluster-api-provider-openstack/values.yaml

```diff
@@ -0,0 +1,6 @@
+configSecret:
+  create: true
```

Comment:
Why do we need to create an empty config?

Reply:
we are using config in [secret.yaml](https://github.com/bnallapeta/hmc/blob/openstack-add-provider/templates/provider/cluster-api-provider-openstack/templates/secret.yaml#L8). If this is not defined, helm template will throw an error. It also lets the users know what fields are available in values.yaml

Reply:
I mean you have empty `.Values.config` by default and thus secret will be created with and empty data (which is weird). So if controller will work fine without the empty secret I suggest setting `configSecret.create: false` and skip the creation of the secret

Reply:
Yes, I did test it with the current way which is an empty config. But I will test this again to make sure it works and then make it false.

Reply:
tested this. it works with `create:false`